microsoft-365/security/office-365-security/install-app-guard.md

```diff
@@ -226,4 +232,5 @@ When this heuristic is met, Office will pre-create an Application Guard containe
 
 * Selecting web links (`http` or `https`) doesn't open the browser.
 * Pasting rich text format (RTF) content or images in Office documents opened with Application Guard isn't supported at this time.
-* Updates to .NET cause files to fail to open in Application Guard. As a workaround, users can restart their device when they come across this failure. Learn more about the issue at [Receiving an error message when attempting to open Windows Defender Application Guard or Windows Sandbox](https://support.microsoft.com/help/4575917/receiving-an-error-message-when-attempting-to-open-windows-defender-ap).
\ No newline at end of file
+* Default setting for unsuported file types protection policy is to block opening untrusted unsupported file types of IRM (Information Rights Management), CSV or HTML
+* Updates to .NET cause files to fail to open in Application Guard. As a workaround, users can restart their device when they come across this failure. Learn more about the issue at [Receiving an error message when attempting to open Windows Defender Application Guard or Windows Sandbox](https://support.microsoft.com/help/4575917/receiving-an-error-message-when-attempting-to-open-windows-defender-ap).
```

Comment:
edit to:

Updates to .NET can cause files to fail to open in Application Guard.

also, can't verify if there is a space between ] and ( for the hyperlink. if there is, kindly remove.